### ISSUE TITLE
Accessibility props

### ISSUE THREAD

**Issue by conor909** [callstack/react-native-paper#2689]
The website homepage says "it supports accessibility standards" & "ships with a lot of components and interactions that are there to satisfy every single use case you might have".

But I only see the `accessabilityLabel` prop on the text component.  Im trying to meet AA standards, but I'm not sure how I would tabindex through the UI with keyboard controls.  It seems I can't pass `focusable` /  `tabindex` & the many other accessibility props I'd want in production. 

I've also checked the source code of some inputs and I dont see how to pass the correct props.  Am I missing something from the docs?

**Comment by lukewalczak** [comment]
Most of the components handle accessibility within itself. Could you please specify with which component you have issues to allow us help you with it?

**Comment by conor909** [comment]
I can pass `tabIndex` as a prop to `TextInputs` by using `inputProps` ie: `<TextInput inputProps={{ tabIndex: '2' }} />`but I don't know how to do that for any other form-controls in my form, Buttons, Checkboxes and RadioButtons.  There is also a list of props with `react-native-web` [in the link](https://necolas.github.io/react-native-web/docs/accessibility/#:~:text=Accessibility%20in%20React%20Native%20for,attributes%2C%20and%20ARIA%20in%20HTML.) that I'd like to pass to components

**Comment by conor909** [comment]
@lukewalczak is there anything further to discuss on this issue?

**Comment by andrejstas** [comment]
@lukewalczak Could you have a look at this issue please?
